src/tools/map_extractor/System.cpp

```diff
@@ -1292,6 +1292,26 @@ void ExtractGameTables()
     printf("Extracted %u files\n\n", count);
 }
 
+
+int GetLocale() {
```

Comment:
Yes, that is true for the rest of the PR too
(general TC recommend C++ codestyle).